docs/sources/source/nlp/ner.rst

```diff
@@ -12,115 +12,120 @@ This tutorial explains how to implement named entity recognition (NER) in NeMo.
 Download Dataset
 ----------------
 
-`CoNLL-2003`_ is a standard evaluation dataset for NER, but any NER dataset will work. The only requirement is that the files are formatted like this:
+`CoNLL-2003`_ is a standard evaluation dataset for NER, but any NER dataset will work. CoNLL-2003 dataset could also be found `here`_. The only requirement is that the data is splitted into 2 files: text.txt and labels.txt. The text.txt files should be formatted like this:
 
 .. _CoNLL-2003: https://www.clips.uantwerpen.be/conll2003/ner/
+.. _here: https://github.com/kyzhouhzau/BERT-NER/tree/master/data
 
 .. code-block::
 
-    Jennifer	B-PER
-    is		O
-    from	O
-    New		B-LOC
-    York	I-LOC
-    City	I-LOC
-    .		O
+    Jennifer is from New York City.
```

Comment:
shouldn't text in text.txt be tokenized? e.g. "City." should be "City ."?

Reply:
fixed